pipelines/zap-ondemand.yml

```diff
@@ -1,20 +1,24 @@
+# local usage:
+#
+#   fly set-pipeline -t lite -n -c pipelines/zap-ondemand.yml -p zap-ondemand -v script-branch=master
+#
 resources:
-- name: scripts 
+- name: scripts
   type: git
   source:
-    uri: {{script-repo}}
-    branch: {{script-branch}} 
+    uri: https://github.com/18F/concourse-compliance-testing.git
```

Comment:
Fair point. I expect we'll want to change the repo name from `concourse-compliance-testing` at some point, but otherwise, it is pretty static. Happy with leaving it hardcoded at the moment.